pkg/webhook/policy.go

```diff
@@ -80,7 +80,7 @@ func init() {
 
 // Explicitly list all known subresources except "status" (to avoid destabilizing the cluster and increasing load on gatekeeper). But include "services/status" for constraints that mitigate CVE-2020-8554.
 // You can find a rough list of subresources by doing a case-sensitive search in the Kubernetes codebase for 'Subresource("'
-// +kubebuilder:webhook:verbs=create;update,path=/v1/admit,mutating=false,failurePolicy=ignore,groups=*,resources=*;pods/ephemeralcontainers;pods/exec;pods/log;pods/eviction;pods/portforward;pods/proxy;pods/attach;pods/binding;deployments/scale;replicasets/scale;statefulsets/scale;replicationcontrollers/scale;services/proxy;nodes/proxy;services/status,versions=*,name=validation.gatekeeper.sh,sideEffects=None,admissionReviewVersions=v1;v1beta1,matchPolicy=Exact
```

Comment:
@JaydipGabani As a separate PR, can you ptal https://github.com/open-policy-agent/gatekeeper/blob/a4a850445bff6be36a0336283080246e64316aa3/pkg/webhook/mutation.go#L52 to make sure it matches deploy/gatekeeper.yaml and helm chart? Looks out of sync.